server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -299,6 +299,7 @@ FastResponseState getFastResponseState() {
     private void handleDisconnectedNode(DiscoveryNode discoveryNode) {
         FollowerChecker followerChecker = followerCheckers.get(discoveryNode);
         if (followerChecker != null) {
+            logger.info(() -> new ParameterizedMessage("{} disconnected", followerChecker));
```

Comment:
It's not consistent throughout this class, but I think it's fine (and more concise) to do

```
logger.info("{} disconnected", followerChecker);
```

like is done on line 433. Feel free to simplify the other lines you modified to this style as well.

Reply:
Changes made as suggested for all the modified logging statements.